src/main/java/org/jabref/gui/openoffice/OOBibBase.java

```diff
@@ -817,9 +835,8 @@ public Optional<BibDatabase> exportCitedHelper(List<BibDatabase> databases, bool
      * @param style     Style.
      */
     public void guiActionUpdateDocument(List<BibDatabase> databases, OOStyle style) {
+        final String errorTitle = Localization.lang("Unable to synchronize bibliography");
```

Comment:
Not sure if this works, I think there was something about using vars for l10n https://devdocs.jabref.org/code-howtos/localization.html#general-hints

Reply:
This existed already (look below at removed line 821).
I can replace the variables with the string, but it will become ugly

Reply:
Seems like this exists in the entire file.